package.json

```diff
@@ -23,7 +23,11 @@
     "format:docs": "cd docs && yarn format && cd ..",
     "find-unused-code:js": "yarn ts-prune --ignore \"index|.web.\" --error  ",
     "type:check": "yarn tsc --noEmit && cd plugin && yarn type:check && cd ..",
-    "type:check-api": "yarn tsc --noEmit --target es6 --module ESNext --jsx react-native --skipLibCheck true --allowSyntheticDefaultImports true --moduleResolution node --esModuleInterop true --strict true --forceConsistentCasingInFileNames true --resolveJsonModule true app/src/App.tsx __typetests__/react-native-reanimated-tests.tsx",
+    "type:run-test": "yarn tsc --noEmit --target es6 --module ESNext --jsx react-native --skipLibCheck true --allowSyntheticDefaultImports true --moduleResolution node --esModuleInterop true --strict true --forceConsistentCasingInFileNames true --resolveJsonModule",
```

Comment:
What's the difference between `type:check` and `type:run-test`? The underlying commands look quite similar, can we also unify their names?